### PR TITLE
circle create page with validators and select features

### PR DESCRIPTION
**Pull request by mzavarka**

### CODE REVIEW

File: apps/schools/domains/circle/components/createCircleForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+                    onValuesChange={validationCheck}
+                    onFinish={() => {
+                        handleSubmitForm(organizationId, form, mutation).then((isSucceed) => {
+                            if (isSucceed) window.location.href = '/circle'
+                        })
+                    }}
+                    layout='vertical'
+                >
+                    <Typography.Title level={1}>Добавление кружка</Typography.Title>
+                    <WithTooltip tooltipText={'Здесь будет текст тултипа'} margin={TOOLTIP_MARGIN}>
```
Comment (levil664) [inline apps/schools/domains/circle/components/createCircleForm/index.tsx:54]: when the text of the tooltip appears?

File: apps/schools/domains/circle/components/createCircleForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                    <Form.Item name='button'>
+                        <Button
+                            disabled={!isFormValid}
+                            key='submit'
+                            type='schoolDefault'
+                            htmlType='submit'
+                            block
+                            data-cy='resetcomplete-button'
+                            style={{ width: '30%' }}
+                            className={styles.button}
```
Comment (levil664) [inline apps/schools/domains/circle/components/createCircleForm/index.tsx]: width must be in className

File: apps/schools/domains/circle/components/createCircleForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                        </Button>
+                    </Form.Item>
+                </Form>
+            </div>
+            <div className={styles.mobileApp}>
+                <Typography.Title level={3}>Приложение родителей</Typography.Title>
+                <Typography.Title level={3} className={styles.text}>
+                    Родители смогут увидеть ваш кружок с помощью карты и узнать информацию о нём!
+                </Typography.Title>
+                {/*<Image src={mobileApp} alt={"Мобильное приложения на странице создания кружка"}/>*/}
```
Comment (levil664) [inline apps/schools/domains/circle/components/createCircleForm/index.tsx]: I think you need to delete

File: apps/schools/domains/circle/components/createCircleForm/styles/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+              border-top-left-radius: 0;
+              border-bottom-left-radius: 0;
+            }
+          }
+        }
+
+        .mapButton {
+          width: 24%;
+          margin-top: 38px;
+          color: #2698FF;
```
Comment (levil664) [inline apps/schools/domains/circle/components/createCircleForm/styles/styles.module.scss]: color there should be in _colors.scss

File: apps/schools/domains/circle/handlers/circle.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import { FormInstance, message } from 'antd'
+import { SuccessCreateCircleMsg } from '@domains/user/components/auth/constants/message'
+import { removeEmpty } from '@domains/common/utils/form'
+import { CIRCLE_NAME, CIRCLE_ADDRESS, ADDRESS_ROOM } from '../components/createCircleForm/constants'
+import { withLoadingMessage } from '@domains/common/utils/loading'
+import { ADDRESS_SEPARATOR } from '@domains/common/utils/geo'
+
+export async function handleSubmitForm(organizationId: string, formComponent: FormInstance, mutation: any) {
+    console.log(formComponent.getFieldValue(CIRCLE_ADDRESS))
```
Comment (levil664) [inline apps/schools/domains/circle/handlers/circle.ts]: delete it

File: apps/schools/domains/circle/handlers/circle.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+    const response = await withLoadingMessage(
+        'Выполняется запрос...',
```
Comment (levil664) [inline apps/schools/domains/circle/handlers/circle.ts]: maybe constant ?

**Review by levil664 — CHANGES_REQUESTED** [review]

File: apps/schools/domains/circle/components/createCircleForm/index.tsx (hunk 1 above)
Comment (mzavarka) [reply apps/schools/domains/circle/components/createCircleForm/index.tsx:54]: on hover

**Pull request merged**